fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/export/checkpoint/ExportCheckpointAlgorithm.java

```diff
@@ -82,18 +101,18 @@ public boolean isReadyToCheckpoint() {
         long cosFileThresholdSize = ConfigurationFactory.getInstance().getCoreCosObjectSizeThreshold();
         long cosMultiPartMinSize = ConfigurationFactory.getInstance().getCoreCosPartUploadTriggerSize();
 
-        // Stop writing to the current COS object
+        // Set to true if we have enough bytes to write a part
+        boolean readyToWrite = isFileExport || (chunkData.getBufferStream().size() > cosMultiPartMinSize);
```

Comment:
This causes a lot of writes to disk in potentially small fragments. Let's get Robin's input.  Writing after every page is what's happening above.

Reply:
I talked with Paul and we think it makes the most sense to introduce a new config parameter that works similar to the cosMultiPartMinSize except for writing to files instead of cos.

Reply:
the new properties are now in place